docs/glossary.md

```diff
@@ -42,6 +42,11 @@ Strategy determines whether children of a given Plan (i.e., Phase) or Phase (i.e
 ### Task
 A Task is a unit of execution of work.
 
+## U
+
+### Unit of Deployment

```

Comment:
I think `Unit of deployment` is just a Pod. Also, `Unit of deployment` is not a first class terminology in SDK docs as yet. So, maybe we don't need to include it here.